.github/workflows/build-zeroconf-broker-container.yaml

```diff
@@ -0,0 +1,29 @@
+name: "Build Zeroconf Protocol Brokers"
```

Comment:
i think for this protocol, we probably want to integrate it more than with our extensibility examples (http, nessie).

to that end, we should update `build/akri-containers.mk` to build the zeroconf broker and create a workflow that uses our actions like `.github/workflows/build-udev-video-broker-container.yml`